index.js

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const { program } = require('commander');
+const mysql = require('mysql2');
+const csvtojson = require('csvtojson');
+const sequelize = require('./dbConnection');
+const Employee = require('./employee');
+const Project = require('./project');
+const ProjectEmployee = require('./projectEmployee');
+const { Op, Model, QueryError } = require('sequelize');
+ const logger = require('../database-cli/utils/logger');
+const { error } = require('console');
+
+
+class QueryWritingError extends Error {
+  constructor(message) {
+    super(message);
+    this.name = "Query Error";
+  }
+}
+
+class SequelizeConnectionError extends Error{
+  constructor(message){
+    super(message);
+    this.name = "Sequelize Error"
+  }
+}
+class DataImportingError extends Error{
+  constructor(message){
+    super(message);
+    this.name = "Importing Error"
+  }
+}
+
+class UnknownError extends Error{
+  constructor(message){
+    super(message);
+    this.name = "Unknown Error"
+  }
+}
+
+// list of employees who are not part of any projects
+program.command("employeeList").description("List of employees who are not part of any projects").action(
+  function(){
+    employeesWithoutProjects()
+    .then((employeeNames) => {
+      logger.info("---------------------------");
+      employeeNames.forEach((employee) => {
+      logger.info(employee.name);
+      });
+      logger.info("---------------------------");
+    })
+    .catch((error) => {
+      logger.error(new QueryWritingError("Unable to fetch employees who are not part of any projects from the database. Please re-check the Sequelize query"));
+    })
+  }
+);
+
+function employeesWithoutProjects() {
+  return new Promise((resolve, reject) => {
+    resolve( employeeNames = Employee.findAll({
+      include: [{
+        model: ProjectEmployee,
+      }],
+      where: {
+        "$ProjectEmployees.projectI$": null
+      },
+    }))
+    reject(error);
+  });
+};
+
+
+
+//list of projects
+program.command('projectList').description("List of projects with information of employee, role, contribution percentage").action(
+  function(){
+    listOfProjects()
+    .then((ProjectNames)=>{
+      logger.info("-----------------------------------------------------------")
+      ProjectNames.forEach((element)=>{
+      logger.info((element.toJSON()).Project.name, " - " + (element.toJSON()).Employee.name, " - ", (element.toJSON()).role, " - ", + (element.toJSON()).contributionPercentage+"%");
+    })
+      logger.info("------------------------------------------------------------")
+    }).catch((error)=>{
+      logger.error(new QueryWritingError("Unable to fetch list of projects from database. Please recheck the query"));
+    })
+  }
+)
+
+function listOfProjects() {
```

Comment:
this file does lot of work, cannwe extract these queries.

Reply:
I have extracted the queries into separate files and used it in the main file by importing.